app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/CheckoutScreen.kt

```diff
@@ -147,7 +192,8 @@ fun CheckoutScreen(
     newPaymentMethodClicked: () -> Unit,
     onDisclaimerItemClicked: (disclaimerItem: DisclaimerItems) -> Unit,
     onAccountabilityLinkClicked: () -> Unit,
-    onChangedPaymentMethod: (StoredCard?) -> Unit = {}
+    onChangedPaymentMethod: (StoredCard?) -> Unit = {},
+    plotEnabled: Boolean = true
```

Comment:
Is plot supposed to be enabled by default here?

Reply:
By default should be false, on the VM if you like @jlplks you can already send for this field the value of the FF, which is should be currently off.

Reply:
Comments addressed